07_procfs/minicom.cap

```diff
@@ -0,0 +1,419 @@
+   15     2 root     SW       0   0%   0% [ksoftirqd/1]
+   28     2 root     SW       0   0%   0% [kdevtmpfs]Mem: 25664K used, 481280K free, 56K shrd, 180K buff, 1868K cached
+CPU:   0% usr   0% sys   0% nic  99% idle   0% io   0% irq   0% sirq
+Load average: 0.00 0.00 0.00 2/82 184
+  PID  PPID USER     STAT   VSZ %VSZ %CPU COMMAND
+  184   180 root     R     1508   0%   0% top
+   30     2 root     RW       0   0%   0% [kworker/1:1-eve]
+  138     2 root     SW       0   0%   0% [xradio_bh]
+   95     2 root     IW       0   0%   0% [kworker/2:2-eve]
+  127     1 root     S     1688   0%   0% /sbin/mdev -df
+  180     1 root     S     1508   0%   0% -sh
+    1     0 root     S     1504   0%   0% init
+  111     1 root     S     1496   0%   0% /sbin/syslogd -n
+  115     1 root     S     1492   0%   0% /sbin/klogd -n
+  179     1 root     S      972   0%   0% /usr/sbin/dropbear -R
+   62     2 root     IW       0   0%   0% [kworker/3:1-mm_]
+   31     2 root     IW       0   0%   0% [kworker/0:1-pm]
+  139     2 root     IW       0   0%   0% [kworker/0:2-eve]
+   60     2 root     IW       0   0%   0% [kworker/u8:1-fl]
+   10     2 root     IW       0   0%   0% [rcu_sched]
+    7     2 root     IW       0   0%   0% [kworker/u8:0-ev]
+   26     2 root     IW       0   0%   0% [kworker/3:0-eve]
+   82     2 root     IW<      0   0%   0% [kworker/3:1H-kb]
+   15     2 root     SW       0   0%   0% [ksoftirqd/1]
+   28     2 root     SW       0   0%   0% [kdevtmpfs]
+# ll
+-sh: ll: not found
+# ls
+# ls -la
+total 3
+drwx------    2 root     root          1024 Jan  1 00:02 .
+drwxr-xr-x   18 root     root          1024 Dec  8  2021 ..
+-rw-------    1 root     root            79 Jan  1 00:08 .ash_history
+# cd /
+# ls
+bin         lib         lost+found  opt         run         tmp
+dev         lib32       media       proc        sbin        usr
+etc         linuxrc     mnt         root        sys         var
+# [  136.837655] random: crng init done
+
+# 
+# addus# adduser vitaliy
+adduser: /home/vitaliy: No such file or directory
+Changing password for vitaliy
+New password: 
+Bad password: too weak
+Retype password: 
+passwd: password for vitaliy changed by root
+# ls
+bin         lib         lost+found  opt         run         tmp
+dev         lib32       media       proc        sbin        usr
+etc         linuxrc     mnt         root        sys         var
+# cd /hshu# sh
+sh         sha1sum    sha256sum  sha3sum    sha512sum  shift      shred
+# shpo# poweroff 
+# Stopping dropbear sshd: OK
+Stopping network: ifdown: interface eth0 not configured
+OK
+Saving random seed: OK
+Stopping mdev... stopped process in pidfile '/var/run/mdev.pid' (pid 127)
+OK
+Stopping klogd: OK
+Stopping syslogd: OK
+umount: devtmpfs busy - remounted read-only
+[  267.919003] EXT4-fs (mmcblk0p2): re-mounted. Opts: (null)
+The system is going down NOW!
+Sent SIGTERM to all processes
+Sent SIGKILL to all processes
+[  269.980873] reboot: System halted
```

Comment:
Wrong output, there is no example output here

Reply:
what is wrong here? this is the console loading orangepi and poweroff

Reply:
Probably you copied wrong output here, there is only boot log password change and poweroff and there is no procfs related logs there.

Reply:
added output from test_module(task07) on orangepi